Benchmarks/Benchmarks/NIOPosixBenchmarks/Benchmarks.swift

```diff
@@ -64,4 +65,43 @@ let benchmarks = {
         )
     }
     #endif
+
+    Benchmark(
+        "MTELG.scheduleTask(in:_:)",
+        configuration: Benchmark.Configuration(
+            metrics: [.mallocCountTotal, .instructions],
```

Comment:
Can we use our `defaultMetrics` here? We don't have instruction based benchmarks yet